types/common.tsx

```diff
@@ -44,6 +44,7 @@ export interface Claim {
   claimDetails?: null | ClaimDetailsResult
   hasPendingWeeks?: null | undefined | boolean
   hasCertificationWeeksAvailable?: null | undefined | boolean
+  hasClaimDetails?: null | undefined | boolean
```

Comment:
This doesn't appear to be used. Can it be removed?

Reply:
its not used in the actual flow, but it supports the api stub and the storybook. If there was some better way to handle this I'm super interested though!

Reply:
removed it in #375